lib/utilities/pristine.js

```diff
@@ -91,7 +91,8 @@ function installPristineApp(appName, options) {
     hasNodeModules,
     hasBowerComponents,
     emberVersion: options.emberVersion || 'canary',
-    emberDataVersion: options.emberDataVersion || 'emberjs/data#master'
+    emberDataVersion: options.emberDataVersion || 'emberjs/data#master',
+    yarn: options.yarn || false
```

Comment:
I am asking myself if it could make sense to not default to `false`, but instead check if the addon has a `yarn.lock` file, and default to `true` in this case? `ember new` needs the `--yarn` option to use yarn, because there is no app yet, but when running `ember install` it does a similar check afaik, assuming it should use yarn if the project uses it. 

Maybe we could copy that behavior here? Fwiw, this could be a separate PR though...